lib/execution/executor.ex

```diff
@@ -894,4 +897,16 @@ defmodule WaspVM.Executor do
 
     Integer.undigits(zero_leading_map ++ bin, 2)
   end
+
+  defp create_entry(instruction) when is_tuple(instruction) == false, do: "#{instruction}"
+  defp create_entry({instruction, variable}), do: "#{instruction}"
+
+  defp write_to_file(instruction, gas) do
```

Comment:
Maybe we shouldn't default to writing this to a file, and instead allow the user to decide what they want to do with this output? It might make sense to create a file-like string (newline delimited) and just spit it out as a return value, at which point the developer can decide if they want to write it to a file.

As an example, I know I'd only use the trace output for debugging, so I'd just want it written to my terminal rather than a file, but other people might want to keep logs, so they'd be able to take the value passed back by trace and write it to a log file

Reply:
Resolved